config.py

```diff
@@ -43,9 +43,7 @@
 # Constants
 # --------------------------------------------------------------------
 
-BAD_SUBJS = {
-    99: "Add new Bad subjs like this (this is an example)",
-}
+BAD_SUBJS = {14: "bad Performance", 31: "bad Performance"}
```

Comment:
```suggestion
BAD_SUBJS = {14: "bad performance, see 00_find_bad_subs.py", 31: "bad performance, see 00_find_bad_subs.py"}
```

also I would prefer `, see 00_find_bad_subjs.py` (with a j, see above :wink:)